src/less/modules/report/report.less

```diff
@@ -0,0 +1,130 @@
+.container {
+  position: relative;
+  display: block;
+  margin: auto;
+  padding-top: 20 px;
+  padding-bottom: 30px;
+  max-width: 1135px;
+  width: 90%;
+
+  &_h1 {
+    font-family: roadgeek;
+    font-weight: 400;
+    line-height: 1.2;
+    text-transform: uppercase;
+    vertical-align: middle;
+    font-size: 2.9rem;
+    color: #000;
+    padding-top: 30px;
+  }
+
+  &_form-font-size {
+    font-size: 18px;
+  }
+
+  &_column_20 {
+    float: left;
+    width: 20%;
+    margin-top: 10px;
+  }
+
+  &_column_25 {
+    float: left;
+    width: 25%;
+    margin-top: 10px;
+  }
+
+  &_column_40 {
+    float: left;
+    width: 40%;
+    margin-top: 10px;
+  }
+
+  &_column_75 {
+    float: left;
+    width: 75%;
+    margin-top: 10px;
+  }
+
+  &_row::after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+}
+
+label {
```

Comment:
Don't apply styles to tag unless you are sure that it won't affect other components on the page.
Please make it with classname instead.